tomviz/LoadDataReaction.cxx

```diff
@@ -181,7 +180,6 @@ DataSource* LoadDataReaction::loadData(const QString& fileName,
 DataSource* LoadDataReaction::loadData(const QStringList& fileNames,
                                        const QJsonObject& options)
 {
-  LoadDataReaction::registerPythonReaders();
```

Comment:
You have to make another call to `FileFormatManager::instance().registerPythonReaders();` 

We could get here without passing through the open file dialog (e.g. from recent files or image stack dialog), so it's important to ensure the python readers are loaded.

(Although since we preemptively register them in the main window we would never really run into issues in practice)